ui/lib/config-ui/addon/components/messages/page/details.js

```diff
@@ -19,17 +19,17 @@ import errorMessage from 'vault/utils/error-message';
  */
 
 export default class MessageDetails extends Component {
-  @service store;
   @service('app-router') router;
   @service flashMessages;
   @service customMessages;
   @service namespace;
+  @service pagination;
 
   @action
   async deleteMessage() {
     try {
-      this.store.clearDataset('config-ui/message');
```

Comment:
typically we call `clearDataset` _after_ deleting so I don't think this was previously working as expected - went ahead and moved method